hail_search/queries/base.py

```diff
@@ -89,9 +89,12 @@ class BaseHailTableQuery(object):
 
     @classmethod
     def load_globals(cls):
-        ht_path = cls._get_table_path('annotations.ht')
-        ht_globals = hl.eval(hl.read_table(ht_path).globals.select(*cls.GLOBALS))
-        cls.LOADED_GLOBALS = {k: ht_globals[k] for k in cls.GLOBALS}
+        try:
```

Comment:
This is probably not the best long term solution (I think we discussed doing a "dataset type/reference genomes" enabled flag), but the service flails in `helm` right now until the pipeline runs and that's not ideal; this at least get the service up.

Reply:
I think it would be better to decide on a plan for explicitly configuring data types rather than have this as the interim solution